frontend/packages/dev-console/src/components/import/pipeline/PipelineTemplate.tsx

```diff
@@ -104,14 +104,7 @@ const PipelineTemplate: React.FC<PipelineTemplateProps> = ({ builderImages }) =>
 
   return pipeline.template ? (
     <>
-      <Flex>
-        <FlexItem breakpointMods={[{ modifier: FlexModifiers['align-self-center'] }]}>
-          <CheckboxField label="Add pipeline" name="pipeline.enabled" />
-        </FlexItem>
-        <FlexItem>
-          <InlineDevPreviewBadge />
-        </FlexItem>
-      </Flex>
+      <CheckboxField label="Add pipeline" name="pipeline.enabled" />
```

Comment:
i see this being added as part of https://github.com/openshift/console/pull/5002/files#diff-d46e40626f43a2fe25c845e59214ab8dR107-R114 So now will we have just TP across pipelines no `devPreview` badge?

Reply:
Correct. Siamak discussed having a singular setting for 4.5. This is not a back-portable change. I thought we'd have them differently but this changed recently after the TP operator went out the door.